executor/main.go

```diff
@@ -173,13 +173,11 @@ func initTracing() io.Closer {
 		cfg.ServiceName = "executor"
 	}
 
-	tracer, closer, err := cfg.NewTracer()
+	propagation := os.Getenv("JAEGER_TRACE_PROPAGATION_TYPE")
```

Comment:
As you have created a new tracing package maybe we should move this function there?